### PR TITLE
language translation

### PR DESCRIPTION
**Pull request by LunaStev**
I translated English README into Korean.

### CODE REVIEW

**Comment by Veykril** [comment]
While having translations for the README could be nice, I don't think we gain much from doing this as the translations will probably become outdated sooner or later. Then again the README isn't really too text heavy either. Any thoughts on this from someone else?

**Comment by lnicola** [comment]
On the other hand, we've had no changes to the README since August (a two character spelling/consistency fix). The previous change is from last April, an URL change. And it seems to me that localization is often _very_ important for people who are not comfortable with English.

If @Quin0019 can squash this and is open to updating the translation if we ping them after a change, I think we could merge it.

File: .README/README-KO.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+러스트 분석기는 러스트 언어를 위한 모듈형 컴파일러 프론트엔드입니다.
+이는 Rust에 대한 우수한 IDE 지원을 만들기 위한 rls-2.0 노력의 일환입니다.
+
+## 빠른 시작
+
+https://rust-analyzer.github.io/manual.html#installation
+
+## 설명
+
+Rust에 **기여하는 방법**이 궁금하다면 [.docs/dev](.docs/dev)폴더 확인하세요.
```
Comment (lnicola) [inline .README/README-KO.md:21]: Wrong link here. It's the same in the original README, but this one points to https://github.com/rust-lang/rust-analyzer/blob/master/.README/.docs/dev. I think it's fine to use a full URL or whatever.

**Comment by lnicola** [comment]
image

image

Huh, can you delete random parts of a sentence in Korean and still get a lossy, but roughly correct result? :joy:.

File: .README/README-KO.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    src="https://raw.githubusercontent.com/rust-analyzer/rust-analyzer/master/assets/logo-wide.svg"
+    alt="rust-analyzer logo">
+</p>
+
+<p align="center">
+  <a href="https://github.com/rust-lang/rust-analyzer/blob/master/README.md">en</a>
+  <a href="#">ko</a>
+</p>
+
+러스트 분석기는 러스트 언어를 위한 모듈형 컴파일러 프론트엔드입니다.
```
Comment (lnicola) [inline .README/README-KO.md:12]: In the Description and License sections you've kept the English/Latin spelling of "rust-analyzer"; same here for "Rust", perhaps. Should it be spelled the same way here? No strong opinion.

**Comment by Veykril** [comment]
Closing this due to inactivity.

**Pull request closed without merging**